### ISSUE TITLE
Will the target detection task code be updated?

### ISSUE THREAD

**Issue by zycer** [sovit-123/dinov3_stack#1]

**Comment by sovit-123** [comment]
Yes @zycer  
I am working on the object detection code at the moment. I have pushed an initial commit to the `detection` branch. You can still run it. However, need some config changes.

In the `src/detection/config.py`, you will need to provide the path to the images and XML annotation folders for training and valdation set, and add the class names. Then you can run the `train_detection.py` script using the command similar to the following:

`python train_detection.py --weight dinov3_convnext_tiny_pretrain_lvd1689m-21b726bb.pth --model-name dinov3_convnext_tiny --imgsz 640 640 --lr 0.0001 --fine-tune --out-dir img_det_aug --epochs 30 --workers 8 --scheduler-epochs 30 --batch 8 `
